src/callout.tsx

```diff
@@ -9,6 +9,7 @@ import IconButton from './icon-button';
 import classNames from 'classnames';
 import {ButtonLink, ButtonPrimary, ButtonSecondary} from './button';
 import ButtonGroup from './button-group';
+import {Inline} from '.';
 
 import type {ButtonProps, ButtonLinkProps} from './button';
 
```

Comment:
this screenshot hasn't a good alignment, right?
https://github.com/Telefonica/mistica-web/pull/414/files#diff-359bb804a1c6f7b7f2063c2a84611a054a27c892c8908dbf0efeb8869a22ce47